lib/livebook.ex

```diff
@@ -130,6 +130,10 @@ defmodule Livebook do
       config :livebook, :force_ssl_host, force_ssl_host
     end
 
+    config :livebook,
+           :embedded_runtime_enabled,
+           Livebook.Config.embedded_runtime_enabled!("LIVEBOOK_EMBEDDED_RUNTIME_ENABLED")
```

Comment:
```suggestion
    config :livebook, :embedded_runtime_enabled, false
```

Reply:
Then, How can I do when enable embedded runtime?

Reply:
Should we keep this variable?
How about remove `config :livebook, :embedded_runtime_enabled, false` and make `config :livebook, :runtime_modules, [Livebook.Runtime.ElixirStandalone, Livebook.Runtime.MixStandalone, Livebook.Runtime.Attached]`? (if not containe `Embedded`, `Embedded` will not connect)

Reply:
Yeah, that's exactly the idea!